entries/c/cpoms.co.uk.json

```
@@ -0,0 +1,17 @@
+{
+  "CPOMS": {
+    "domain": "cpoms.co.uk",
+    "additional-domains": [
+      "cpoms.net"
+    ],
+    "img": "cpoms.co.uk.png",
+    "tfa": [
+      "totp",
+      "u2f"
+    ],
+    "documentation": "https://raptor.my.site.com/CPOMS/s/article/CPOMS-Login-Help-Guide-4cb23e40",
+    "categories": [
+      "other"
+    ]
```

It seems to me that this is a UK-only service, especially given the use of a `.co.uk` [ccSLD](https://www.dynadot.com/community/help/question/what-is-ccsld) for their primary domain.

```suggestion
    ],
    "regions": [
      "gb"
    ]
```